.github/workflows/php.yml

```diff
@@ -25,6 +25,9 @@ jobs:
               restore-keys: |
                   ${{ runner.os }}-composer-
 
+      -   name: Validate composer.{json,lock} files
+          run: composer validate --strict
+
```

Comment:
This shouldn't be part of PHP CS fixer, but a separate job instead.

Reply:
I can do that, but I do not think extra job is good for this small check.

PS Maintaners, advise, I can do that if you want.